app/src/main/java/com/example/next_show/MainActivity.java

```diff
@@ -37,4 +41,11 @@ private void setUpNavigation() {
         // connect the bottom view with the controller
         NavigationUI.setupWithNavController(bottomNavigationView, navController);
     }
+
+    // widely used function to make Toast showing easier for fragments in MainActivity
+    public static void showToast(String m, Context context) {
+        Toast msg = Toast.makeText(context, m, Toast.LENGTH_SHORT);
+        msg.setGravity(Gravity.TOP|Gravity.CENTER_HORIZONTAL, 0, 0);
+        msg.show();
+    }
```

Comment:
Might make sense to move this out into a ToastUtil class instead and then can name the function `showTopAlignedToast()`